app/views/home/home.html.erb

```diff
@@ -1,4 +1,208 @@
 <div class="container-fluid">
```

Comment:
`<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">`
Replace with this